forms/mainwindow.ui

```diff
@@ -3034,6 +3040,7 @@
    <class>AdjustingStackedWidget</class>
    <extends>QStackedWidget</extends>
    <header>adjustingstackedwidget.h</header>
+   <container>1</container>
```

Comment:
Any idea what this `<container>1</container>` change is?

Reply:
No clue.

Reply:
I've noticed Qt Designer will add that (not sure when or why), I've always removed it manually since I didn't know what it was, and it wasn't present in any of the other porymap `.ui` files that I looked at.